recipe/meta.yaml

```
@@ -27,36 +27,38 @@ outputs:
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
         - numpy                                  # [build_platform != target_platform]
+        - meson-python>=0.13.1                   # [build_platform != target_platform]
         - pybind11 >=2.6                         # [build_platform != target_platform]
-        - pkg-config  # [not win]
+        - pkg-config
```

It looks like `meson` is searching for `cmake` on Windows [on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=963332&view=logs&j=00f5923e-fdef-5026-5091-0d5a0b3d5a2c&t=f42c1d49-d161-5453-1c0e-990f827a8536&l=871 )

```
Found CMake: C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.EXE (3.28.3)
```

Should we add it here?

```suggestion
        - cmake                                  # [win]
        - pkg-config
```

---

It's only looking for CMake because it can't find FreeType via pkg-config; is that not correctly available for FreeType on Windows?

---

We [build `freetype` with `cmake` on Windows]( https://github.com/conda-forge/freetype-feedstock/blob/536399f6b9216fd930c26caf90319df7521d0d8f/recipe/bld.bat#L8 ). So yeah using `cmake` sounds desirable

Guessing the `cmake` builds of `freetype` don't necessarily install `.pc` files (particularly not on Windows)

---

OK, I've never tried finding FreeType via CMake, so hopefully the dependency names are right.

---

Interesting ok. Wondering how this was working before this update

---

Yep we need `pkg-config`. Though it looks like this may work. More details here: https://github.com/conda-forge/matplotlib-feedstock/pull/373#discussion_r1655870742

Trying in commit: https://github.com/conda-forge/matplotlib-feedstock/pull/373/commits/d8db30f329e5a01913ee1f036f371fd24efe1734